code/renderers/web-components/src/docs/web-components-properties.test.ts

```diff
@@ -11,10 +11,14 @@ const inputRegExp = /^input\..*$/;
 const runWebComponentsAnalyzer = (inputPath: string) => {
   const { name: tmpDir, removeCallback } = tmp.dirSync();
   const customElementsFile = `${tmpDir}/custom-elements.json`;
-  spawnSync('yarn', ['wca', 'analyze', inputPath, '--outFile', customElementsFile], {
-    stdio: 'inherit',
-    shell: true,
-  });
+  spawnSync(
+    path.join(__dirname, '../../../../node_modules/.bin/wca'),
```

Comment:
Why this change?  It feels a bit fragile to provide a direct path like this, though in reality it's probably fine.

Reply:
I'm not sure what exactly caused it to break, but jest/yarn couldn't find the binary anymore, and this fixed it.

I shortly tried to convert it to using the node API, but it seems the output of the nodeAPI is completely different..
I gave up after a while.

I also considered just checking in the generated code, and not running this generator at all anymore, but there's likely a reason this is done that way? Maybe it's to detect changes in the behavior of that package upstream? All seems pretty fragile already to me...